measures/HPXMLtoOpenStudio/measure.rb

```diff
@@ -3028,6 +3029,51 @@ def self.add_output_variables(model, vars, objects)
     end
   end
 
+  def self.add_ems_cooling_heating_load_output(model)
+    control_zones = @control_slave_zones_hash.keys
```

Comment:
Need to change this line/part when single thermal zone approach is implemented.

Reply:
The CI will very loudly complain if it's not updated, because `@control_slave_zones_hash` won't exist anymore.

Reply:
Could imagine 😵